addons/base_address_extended/models/res_partner.py

```diff
@@ -17,123 +13,39 @@ class Partner(models.Model):
     street_number2 = fields.Char(
         'Door', compute='_compute_street_data', inverse='_inverse_street_data', store=True)
 
+    city_id = fields.Many2one('res.city', string='City')
+    country_enforce_cities = fields.Boolean(related='country_id.enforce_cities', readonly=True)
```

Comment:
by default, a `related` field is `readonly` so you can drop the last parameter

Reply:
done